cves/CVE-2010-1452.yml

```diff
@@ -204,4 +212,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This issue seems attributable to a coding mistake, as the implementation
+    of request parsing made an assumption.
```

Comment:
Maybe elaborate more on this. It seems a little short.

Reply:
Expanded on it a little bit - didn't want to get too technical.